components/NewRepoModal.js

```diff
@@ -6,13 +6,22 @@ import {
   StyleSheet,
```

Comment:
Guess this component shouldn't be a modal anymore. It doesn't make sense since it is on a differente screen now. Change its name to `NewRepoForm`, for example, makes way more sense.